deployment/constructs/lambda.ts

```diff
@@ -14,18 +14,16 @@ import {
   ServicePrincipal,
 } from "aws-cdk-lib/aws-iam";
 import * as apigateway from "aws-cdk-lib/aws-apigateway";
+import { LogGroup, RetentionDays } from "aws-cdk-lib/aws-logs";
 import { isLocalStack } from "../local/util";
 
 interface LambdaProps extends Partial<NodejsFunctionProps> {
-  handler: string;
-  timeout?: Duration;
-  memorySize?: number;
```

Comment:
Because of `Partial<NodejsFunctionProps>` we don't need to define things that are in `NodejsFunctionProps`